diagnostic.md

```diff
@@ -5,28 +5,29 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+    Something that has elements nested within it, like a navbar with buttons or input fields.
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember generate component my-map
     ```
 
 1.  What files are edited to produce a component, and what are their
     responsibilities?
 
     ```md
-    <!-- your response here -->
+    Templates (template.hbs) and components (components.js).  Components define the actions and behavior of templates using JS and populate the model via the model hook.  Templates contain the HTMLbars content that will be displayed for that particular section.
     ```
 
 1.  Suppose you have a component '`my-contact`', which is loaded from
     '`app/contacts/template.hbs`' when visiting the `/contacts` path. What is
     the syntax for loading this component inside that template?
 
     ```html
-    <!-- your response here -->
+    {{ each contact as |contact| }}
```

Comment:
you have to assign the model to contact for semantic iteration, then bind contact as contact

{{#each model as |contact|}}
{{my-contact contact=contact}}
{{/each}}